docs/users/customization-extendibility.md

```diff
@@ -5,28 +5,41 @@ For most standard web applications, ddev provides everything you need to success
 The majority of ddev's customization ability and extensibility comes from leveraging features and functionality provided by [Docker](https://docs.docker.com/) and [Docker Compose](https://docs.docker.com/compose/overview/). Some working knowledge of these tools is required in order to customize or extend the environment ddev provides.
 
 ## Extending with additional docker-compose files
-Under the hood, ddev uses docker-compose to define and run the multiple containers that make up the local environment for a web application. Docker compose supports defining multiple compose files to facilitate [sharing Compose configurations between files and projects](https://docs.docker.com/compose/extends/), and ddev is designed to leverage this ability. Additional services can be added to your project by creating additonal docker-compose files in the `.ddev` directory for your project. Any files using the `docker-compose.[servicename].yml` naming convention will be automatically processed by ddev to include them in executing docker-compose functionality. A `docker-compose.override.yml` can additonally be created to override any configurations from the main docker-compose file or any service compose files added to your project. 
+Under the hood, ddev uses docker-compose to define and run the multiple containers that make up the local environment for a web application. Docker compose supports defining multiple compose files to facilitate [sharing Compose configurations between files and projects](https://docs.docker.com/compose/extends/), and ddev is designed to leverage this ability. Additional services can be added to your project by creating additional docker-compose files in the `.ddev` directory for your project. Any files using the `docker-compose.[servicename].yml` naming convention will be automatically processed by ddev to include them in executing docker-compose functionality. A `docker-compose.override.yml` can additionally be created to override any configurations from the main docker-compose file or any service compose files added to your project.
 
 ## Conventions for defining additonal services
 When defining additional services for your project, it is recommended to follow these conventions to ensure your service is handled by ddev the same way the default services are.
+
 - Containers should be follow the naming convention `local-[sitename]-[servicename]`
+
 - Containers should be provided the following labels:
+
   - `com.ddev.site-name: ${DDEV_SITENAME}`
```

Comment:
I can see the argument both ways. This feels like example configuration. I do think we should probably decide where and how we use backticks and code blocks in docs and try to standardize usage, though.

This probably points at a higher level need to develop some sort of style guide for docs before too long.